node-tests/acceptance/build-test.js

```diff
@@ -66,6 +66,7 @@ describe('Acceptance', function() {
 
         let output = await build(app);
 
+        debugger;
```

Comment:
✂️

Reply:
ya, currently fixing this test scenario. I'll remove, as it didn't need to actually be committed.